pkg/apis/k8s/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -86,6 +90,12 @@ type NginxIngressControllerSpec struct {
 	// – only one replica will report status.
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	IngressLink string `json:"ingressLink,omitempty"`
+	// Specifies the name of the IngressLink resource, which exposes the Ingress Controller pods via a BIG-IP system.
+	// The IP of the BIG-IP system is used when reporting the status of Ingress, VirtualServer and VirtualServerRoute resources.
+	// For Ingress resources only: Requires -report-ingress-status.
```

Comment:
should we refer to fields of the NginxIngressController resource, not cli arguments ?